client/index.html

```diff
@@ -18,48 +18,30 @@
         <!-- Angular Material Library -->
         <script src="http://ajax.googleapis.com/ajax/libs/angular_material/1.1.0/angular-material.min.js"></script>
         <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+        <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.4.3/angular-route.js"></script>
         
-        <script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.9.0/moment.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/angular-moment/0.9.0/angular-moment.min.js"></script>
+        <script src="http://cdnjs.cloudflare.com/ajax/libs/moment.js/2.9.0/moment.min.js"></script>
+        <script src="http://cdnjs.cloudflare.com/ajax/libs/angular-moment/0.9.0/angular-moment.min.js"></script>
+
         <!-- APP -->
-        <script src="js/app.js"></script>
+        <script src="angular/app.js"></script>
 
         <!-- CONTROLLERS -->
-        <script src="js/controllers/MainController.js"></script>
+        <script src="angular/controllers/MainController.js"></script>
+        <script src="angular/controllers/AdminController.js"></script>
 
         <!-- SERVICES -->
-        <script src="js/services/twitterWallDataService.js"></script>
+        <script src="angular/services/twitterWallDataService.js"></script>
+        <script src="angular/services/adminDashDataService.js"></script>
+
+        <!-- CONFIG -->
+        <script src="angular/config/routeConfig.js"></script>
         
     </head>
     <body ng-controller="MainController">
```

Comment:
Is this line needed if the controller is placed in by the route? Otherwise looks all good.